languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -90990,21 +91003,21 @@ The accident victim died from her injuries.
             <token>tummy</token>
             <token regexp="yes">aches?</token>
         </pattern>
-        <message>In some contexts, this is childish language. Consider alternatives.</message>
+        <message>In some contexts, '\1' is childish language.</message>
         <suggestion>stomach \2</suggestion>
         <example correction="stomach ache">I have <marker>tummy ache</marker> because I ate too much candy.</example>
     </rule>
     <rule>
         <pattern>
             <token regexp="yes">tummyaches?</token>
         </pattern>
-        <message>In some contexts, this is childish language. Consider alternatives.</message>
+        <message>In some contexts, '\1' is childish language.</message>
```

Comment:
This one is a pet peeve of mine. I'd go about it even more subtly — ..., '\1' might be considered childish.
In French, you have the word "usage familial" for words that are usually family-internal. Is there any comparable alternative to "childish"? If find that word pretty insulting, frankly.